thali/NextGeneration/thaliPeerPool/thaliPeerPoolDefault.js

```diff
@@ -63,11 +64,18 @@ var ForeverAgent = require('forever-agent');
  */
 function ThaliPeerPoolDefault() {
   ThaliPeerPoolDefault.super_.call(this);
+  this.stopped = false;
```

Comment:
There is another variable with the same sense in project/ Its name is '_started'. We can use '_stopped' here.